src/main/java/echoquery/sql/QueryBuilder.java

```diff
@@ -84,17 +92,11 @@ public static QueryBuilder of(Intent intent) {
     } else {
       aggregationFunc = QueryUtil.functionCall(
           SlotUtil.getAggregateFunction(aggregate),
-          QueryUtil.nameReference(aggregationColumn));
+          new QualifiedNameReference(
+              QualifiedName.of(from.wherePrefix(1), aggregationColumn)));
```

Comment:
like we call
    infer(table, comparisonColumn, aggregationColumn);

so to get the prefix of comparisonColumn you ask for wherePrefix(0)